raco/myrial/interpreter.py

```diff
@@ -168,14 +167,17 @@ def extract_unbox_args(self, from_args, sexpr):
                 if name not in from_args:
                     from_args[name] = self.__lookup_symbol(name)
 
-    def bagcomp(self, from_clause, where_clause, emit_clause):
+    def bagcomp(self, from_clause, where_clause, emit_clause,
+                orderby_clause, limit_clause):
         """Evaluate a bag comprehension.
 
         from_clause: A list of tuples of the form (id, expr).  expr can
         be None, which means "read the value from the symbol table".
 
         where_clause: An optional scalar expression (raco.expression).
 
+        orderby_clause: An optional list of OrderbyArg instances.
```

Comment:
need a docstring for limit_clause, also docstrings should be in same order as args